### DISCUSSION TITLE
Is it possible to exclude a field from a superclass when inheriting PydanticModel?

### DISCUSSION THREAD

**Discussion by vpatov** [strawberry-graphql/strawberry#2510] [Q&A]
Sample code:

```python3
class Parent(PydanticModel):
    a: str
    b: str
    c: str
    exclude_me: str


class Child(Parent):
    d: str
    # I want to get rid of "exclude_me" entirely, as if it were never defined in the parent. (it being None is not good enough)
```

Is there a way I can achieve this?

I also don't strictly need to use inheritance - perhaps there is a way where I could create a model using a list of fields from the parent and exclude just one field?

**Comment by ansarsk** [comment]
how about using dataclass ?

exlude_me: str  = field(repr=False) # omits this field
